lib/tasks/data/cities.yml

```diff
@@ -101,7 +101,7 @@
 - Warrington
 - Watford
 - West Bromwich
-- Weston-super-Mare
+- Weston-Super-Mare
```

Comment:
Why dashes are accepted here?

Reply:
There's dashes in the name in the API response.

Reply:
I capitalised the S because I'm extra. Doesn't make a difference.